### PR TITLE
matplotlib-venn 0.11.10

### PR DESCRIPTION
**Pull request by psteyer**
matplotlib-venn 0.11.10 :snowflake:

**Destination channel:** Snowflake 

### Links

- [PKG-5100](https://anaconda.atlassian.net/browse/PKG-5100) 
- [Upstream repository](https://github.com/konstantint/matplotlib-venn/tree/0.11.10)
- [Upstream changelog/diff](https://github.com/konstantint/matplotlib-venn/compare/0.11.7...0.11.10)

### Explanation of changes:

- build for 0.11.10
- changing source in order to include tests
- removing test for py38. See thread for details

[PKG-5100]: https://anaconda.atlassian.net/browse/PKG-5100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lorepirri — APPROVED** [review]
Nice one! Thanks for the tests!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
-  sha256: 8d8b3505594313099328c9f177133dd800a28f3dd68e6800a78250c0cccce317
+  url: https://github.com/konstantint/matplotlib-venn/archive/refs/tags/{{ version }}.tar.gz
+  sha256: acb254f6fc0c8eb4e49df6655b316903d6babe2e9c0049636c2426a3a861cfbc
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-deps --no-build-isolation
+  skip: True  # [py<36]
```
Comment (ifitchet) [inline recipe/meta.yaml]: There's a `noarch` line and you have two `--no-deps`

**Review by ifitchet — COMMENTED** [review]
Also "delete branch on merge" and "branch protection rules" :)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,53 @@
...
 
 source:
-  url: https://pypi.org/packages/source/m/matplotlib-venn/matplotlib-venn-{{ version }}.tar.gz
-  sha256: 8d8b3505594313099328c9f177133dd800a28f3dd68e6800a78250c0cccce317
+  url: https://github.com/konstantint/matplotlib-venn/archive/refs/tags/{{ version }}.tar.gz
+  sha256: acb254f6fc0c8eb4e49df6655b316903d6babe2e9c0049636c2426a3a861cfbc
 
 build:
   number: 0
   noarch: python
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,39 +1,53 @@
...
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-deps --no-build-isolation
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: good catch, I've added suggestions

**Comment by psteyer** [comment]
I'm unsure why all the sudden `linux-64` is having a test failure. It's specifically for python 3.8 I believe.  `FAILED tests/region_test.py::test_circle_region`. 

I'm not sure I understand the reason for the `Assertion Error`. In this case should I just go ahead and skip `test_circle_region`?

**Comment by skupr-anaconda** [comment]
> I'm unsure why all the sudden `linux-64` is having a test failure. It's specifically for python 3.8 I believe. `FAILED tests/region_test.py::test_circle_region`.
> 
> I'm not sure I understand the reason for the `Assertion Error`. In this case should I just go ahead and skip `test_circle_region`?

@psteyer Yes, you can skip `test_circle_region` for **py38** because it requires sufficient precision on CI systems, see the upstream issue https://github.com/konstantint/matplotlib-venn/issues/76 and the maintainers also skip testing it https://github.com/konstantint/matplotlib-venn/blob/0.11.10/.travis.yml#L7-L8.
It can be an issue caused by numpy 1.24.3 (?) and python 3.8 on linux-64.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,39 +1,52 @@
...
 about:
   home: https://github.com/konstantint/matplotlib-venn
   license: MIT
   license_file: LICENSE
   license_family: MIT
   summary: Functions for plotting area-proportional two- and three-way Venn diagrams in matplotlib.
+  description: |
+    Routines for plotting area-weighted two- and three-circle venn diagrams.
   dev_url: https://github.com/konstantint/matplotlib-venn
+  doc_url: https://github.com/konstantint/matplotlib-venn/blob/master/README.rst
```
Comment (ifitchet) [inline recipe/meta.yaml]: If you're going to use the default branch's README then you don't need to point into `/blob/`.

```suggestion
  doc_url: https://github.com/konstantint/matplotlib-venn
```

**Review by ifitchet — APPROVED** [review]

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,39 +1,53 @@
...
 
 test:
+  source_files:
+    - tests
   imports:
     - matplotlib_venn
+  commands:
+    - pip check
+    - pytest -v . -k "not test_circle_region" # [py38]
+    - pytest -v .                             # [not py38]
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
    - pytest -v . -k "not test_circle_region" # [py==38]
    - pytest -v .                             # [not py==38]
```

**Pull request merged**